data/scripts.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+const {PRNG} = require('./../prng');
```

Comment:
Rather minimal. Random Battle stopped working for 10ish minutes. Not our worst outage.